js/ts/chocolatey-tab-multiples.ts

```diff
@@ -0,0 +1,67 @@
+import { Tab } from 'bootstrap';
+import { getCookie, setCookieExpirationNever } from './util/chocolatey-functions';
+
+(() => {

```

Comment:
Assuming I'm understanding what's going on here: line 4 is creating an anonymous function (or lambda, or script block, or however you might refer to it in TypeScript 😂 ), then on line 67 it's calling said function. Is there any reason not to just call have the code in the file? Is it because of the way the browser processes it?

Reply:
Nice. I had thought maybe it was something to do with the DOM being loaded, but obviously wasn't that (although is now 😄 ). Hadn't thought of variable hiding.